samples/chat/java/local.settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+    "FUNCTIONS_WORKER_RUNTIME": "java"
+  }
+}
```

Comment:
do java function apps make use of funcignore file to ignore certain files in the deployments?
If so, please add it.